src/home_robot_hw/home_robot_hw/remote/api.py

```diff
@@ -153,3 +157,34 @@ def dpt_cam(self):
     def get_frame_pose(self, frame, base_frame=None, lookup_time=None):
         """look up a particular frame in base coords"""
         return self._ros_client.get_frame_pose(frame, base_frame, lookup_time)
+
+    def go_to_manip_mode(self):
+        """Move the arm and head into manip mode."""
+        """
+        home_q = STRETCH_PREGRASP_Q
+        home_q = self.robot_model.update_look_at_ee(home_q)
+        self.goto(home_q, move_base=False, wait=True)
+        """
+        self.switch_to_manipulation_mode()
+        self.head.look_at_ee(blocking=False)
+        self.manip.goto_joint_positions(
+            self.manip._extract_joint_pos(STRETCH_PREGRASP_Q)
+        )
+        self.switch_to_navigation_mode()
```

Comment:
Should we be switching to navigation mode in the `go_to_manip_mode` function?

Reply:
yeah probably not necessary